wandb/util.py

```diff
@@ -826,6 +841,11 @@ def json_dump_uncompressed(obj: Any, fp: IO[str], **kwargs: Any) -> None:
     return dump(obj, fp, cls=JSONEncoderUncompressed, **kwargs)
 
 
+def json_dumps_weave(obj: Any, **kwargs: Any) -> str:
```

Comment:
not: This is a misnomer, there is not a weave part of this

Reply:
I added code to check if a `Media` type has `.to_weave` defined on it.  Currently none do, but I was thinking we would probably want to keep the serialization logic on those objects when we support them.